src/app/modules/main/wallet_section/transactions/module.nim

```diff
@@ -59,21 +58,18 @@ method isLoaded*(self: Module): bool =
   return self.moduleLoaded
 
 method viewDidLoad*(self: Module) =
-  self.checkRecentHistory()
+  self.controller.checkRecentHistory(true)
```

Comment:
Can you add the param name in front, just for clarity. eg: `(visible = true)`

Reply:
aah sure!